revscoring/features/revision.py

```diff
@@ -45,6 +71,19 @@ def process_has_custom_comment(revision_metadata):
 has_custom_comment = Feature("revision.has_custom_comment",
                              process_has_custom_comment,
                              returns=bool, depends_on=[revision.metadata])
+"""
+Represents whether the edit has custom edit summary.
```

Comment:
However, users are also allowed to create a custom summary with more than one of `/* these */`...